### ISSUE TITLE
Twitter/Codebird not working

### ISSUE THREAD

**Issue by kamstutz** [pavelk2/social-feed#276]
It appears that the Codebird Twitter integration for SocialFeed has broken.  Multiple javascript errors have started to appear all related to codebird.  Such as is shown below.

**https://api.jublo.net/codebird/oauth2/token 502 (Bad Gateway)
XMLHttpRequest cannot load https://api.jublo.net/codebird/oauth2/token. Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.**

**Comment by HartLarsson** [comment]
i've just chekced and for me twitter feed works perfectly. there was an error some weeks ago due to their platform update but now is working again.
Please verify you still using the latest script version and the right codebird version too

**Comment by mutiullah7** [comment]
yup it's working fine for now but sometimes twitter posts show up sometimes not. But mostly it shows the twitter posts. i think it's okay

**Comment by mutiullah7** [comment]
@kamstutz It's not working now, giving error 
Failed to load resource: net::ERR_CONNECTION_TIMED_OUT        api.jublo.net/codebird/oauth2/token

**Comment by HartLarsson** [comment]
is not a social-feed script problem, is related to codebird API.
I suggest to open a ticket there

**Comment by mutiullah7** [comment]
@HartLarsson okay thanks :)

**Comment by mutiullah7** [comment]
Twitter Feed again stop working
`OPTIONS https://api.jublo.net/codebird/oauth2/token net::ERR_CONNECTION_TIMED_OUT`

**Comment by Hypocrite** [comment]
Confirmed. Its broken again. Can we change to some other service which is more reliable?

**Comment by chrismwiggs** [comment]
Confirming broken again. Might be worth thinking about alternatives!

**Comment by HartLarsson** [comment]
If someone know a valid opensource alternative to codebird could post here the URL of the project to take a look

**Comment by HartLarsson** [comment]
anyway seems is working again

**Comment by himankpathak** [comment]
I was facing the same issue with Twitter API, I updated the codebird API to the latest version and it started to work. We should upgrade the codebird API of the project.

**Comment by mutiullah7** [comment]
@HartLarsson may be this one? https://github.com/jasonmayes/Twitter-Post-Fetcher

**Comment by mutiullah7** [comment]
Stopped working again :(
